arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,139 @@
+
+## In a nutshell:
+
+- BOLD, an acronym for Bounded Liquidity Delay, is a new challenge protocol for Arbitrum chains that ensures disputes will take, at most, 1 challenge period (6.4 days) to get resolved. BOLD effectively eliminates delay attacks by enforcing a upper time-bound for dispute resolution.
+- Validation for Arbitrum One and Nova is currently is limited to a [permissioned set of parties maintained by the Arbitrum DAO](https://docs.arbitrum.foundation/state-of-progressive-decentralization#allowlisted-validators) to reduce the risks of *[delay attacks](https://medium.com/offchainlabs/solutions-to-delay-attacks-on-rollups-434f9d05a07a)* where malicious entities can indefinitely open disputes and delay confirmations of assertions about Arbitrum states.
+- However, BOLD mitigates the risks for delay attacks and therefore paves the way for permissionless validation of Arbitrum chains. Permissionless validation is a key milestone on [Arbitrum’s journey to becoming a Stage 2 Rollup](https://docs.arbitrum.foundation/state-of-progressive-decentralization) - the most advanced and mature rollup technology categorization. With BOLD, **any honest party can stake on the correct L2 state and will always win disputes against malicious entities.**
+- BOLD is currently considered to be in `alpha` release and is deployed on a public testnet. [Follow this guide](https://github.com/OffchainLabs/bold-validator-starter-kit) to deploy a BOLD validator to test & explore, first hand, how BOLD works to secure Arbitrum chains. To learn more about BOLD, please check out the [BOLD whitepaper](https://github.com/OffchainLabs/bold/blob/main/docs/research-specs/BOLDChallengeProtocol.pdf) and [BOLD's code and specifications on Github](https://github.com/OffchainLabs/bold).
+
+## What _exactly_ is BOLD?
+BOLD, an acronym for Bounded Liquidity Delay Protocol, is a set of rules used by validators to open and resolve disputes about Arbitrum’s state to ensure only valid states get confirmed to an Arbitrum chain’s data availability layer, such as Ethereum, for example. BOLD is shipped as part of the Nitro tech stack - the software that powers every Arbitrum validator node. 
+
+In short, BOLD secures the process of withdrawing assets from Arbitrum back to Ethereum.
+
+A BOLD-enabled validator’s responsibilities are to:
+
+- Post claims about an Arbitrum chain’s state to Ethereum,
+- Challenge invalid claims made by other validators, and
+- Confirm valid claims - either by timing other validators out or by winning a challenge
+
+The goal of BOLD is ensure the maximum challenge period is 6.4 days, effectively removing the risk of delay attacks and making L2 to L1 withdrawals more secure. BOLD accomplishes this by introducing a new dispute system that lets a single entity secure Arbitrum against malicious parties - effectively allowing anyone to validate Arbitrum’s state without needing permission to do so.
```

Comment:
Here too, the maximum challenge period is 2*6.4

Reply:
Also, we might want to change the "L2 to L1" wording to something like "withdrawals to the parent chain", since this will also be applied to Orbit chains.

Is a bit tricky, since we clearly want to emphasize the fact that Arbitrum chains are one of the main beneficiaries of this update, but we probably want to make sure Orbit chains are also benefited from this.

Another thing we could do is to add a note at the end or the beginning of the page stating that although "L2 to L1" language is used, this also applies to any chain settling to a parent chain.

Reply:
Finally, "a new dispute system that lets a single entity secure Arbitrum", this is the case with the current challenge protocol too, is not something new with Bold. Is one of the main selling points of the current challenge protocol, so I think we shouldn't mention it as a feature of Bold.